engines/bastion_katello/app/assets/stylesheets/bastion_katello/host_subscriptions.scss

```diff
@@ -0,0 +1,16 @@
+table.table-bordered {
+  tbody {
+    tr.row-selector-label {
+      td {
+        &.row-select {
+          border-right: 0;
+        }
+        &:last-of-type {
+          border-left: 0;
+          position: relative;
+          left: -30px;
```

Comment:
@ehelms I'm trying to use the $row-select-width variable from [here](https://github.com/Katello/bastion/pull/178/files) but  I get an error when I use it. Do I have to import something to use bastion SASS variables?